src/devtoolsPanel.ts

```diff
@@ -414,6 +426,17 @@ export class DevToolsPanel {
             `;
     }
 
+    private setBrowserRevision(revision: string) {
+        if (revision !== '') {
+            this.config.isCdnHostedTools = true;
```

Comment:
NIT: If the devtoolsBaseUri is set in the config, can we check for it and avoid setting up the BrowserVersionDetectionSocket / the version detection logic altogether?

Reply:
Good call. I'll work on this in the morning. I think I might make separate variables for some of these and make config readonly to make sure we don't override things unintentionally.

Reply:
Had a quick offline discussion with Michael. I am going to create separate variables to prevent writing the config object within devtools panel. However, I'm not going to skip the version detection logic as it is needed in the follow up headless pr for now. Since this bypass would only apply to local debugging, and could potentially cause confusion when testing headless settings, etc, we thought it would be best to not do that part.